### DISCUSSION TITLE
[bug] Upgrade to 6.5 - verification email not arriving

### DISCUSSION THREAD

**Discussion by CHILLTH** [YetiForceCompany/YetiForceCRM#17348] [General]
Upgraded system, tried verify email, but code not arriving in mailbox. Cannot re-register installed CRM

Best

**Comment by CHILLTH** [comment]
Moreover... registration modal make it impossible to carryon work.

**Comment by rskrzypczak** [comment]
did you check your spam folder? maybe it went there

**Reply by CHILLTH** [reply]
yes, used two various email addresses. No verification code arrived

**Comment by paula-w** [comment]
Can you try requesting the code once again?
